packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue

```diff
@@ -336,25 +336,8 @@ export default defineComponent({
       return this.$route.params.storageId || null
     }
   },
-  watch: {
-    highlightedFile(newItem, oldItem) {
```

Comment:
Why can the watcher be removed?

Reply:
its move up into the parent component. This saves one additional request (Shares, Links)